docs/resources/azure_monitor_activity_log_alerts.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the azure_monitor_activity_log_alerts Resource
+platform: azure
+---
+
+# azure\_monitor\_activity\_log\_alerts
+
+Use the `azure_monitor_activity_log_alerts` InSpec audit resource to verify an Activity Log
+Alert exists.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_activity_log_alerts` resource block identifies a Activity Log Alerts by name.
```

Comment:
Use a/an if keeping singular, or drop the article if you mean to keep the plural.

Why don't we see `name` in the example below?

Reply:
`name` isnt required for this since you would use it to search all activity log alerts to see if one is there with a given name.